src/main/kotlin/com/exactpro/th2/act/core/messages/MessageBuilder.kt

```diff
@@ -17,15 +17,18 @@
 package com.exactpro.th2.act.core.messages
 
 import com.exactpro.th2.common.grpc.*
+import com.exactpro.th2.common.message.toTimestamp
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.BookName
 import com.exactpro.th2.common.value.nullValue
 import com.exactpro.th2.common.value.toValue
+import java.time.Instant
 
 @DslMarker
 annotation class MessageBuilderMarker
 
 fun main() {
     val connectionID = ConnectionID.getDefaultInstance()
-    message("MessageType", connectionID) {
+    message("MessageType", connectionID, "book") {
```

Comment:
This method looks strange because it create dummy message with fake values. I hope it isn't used in our implementations

Reply:
Please, just remove the main method. It was left as an example